packages/reactotron-server/src/server/datastore/commands.ts

```diff
@@ -7,8 +7,22 @@ export class Commands {
     this.commands.push(command)
   }
 
-  byClientId(clientId) {
-    return this.commands.filter(cmd => cmd.clientId === clientId)
+  filterItem(item: Command, clientId?: string, filter?: [string]) {
+    return (
+      (!clientId || item.clientId === clientId) &&
+      (!filter || filter.some(fil => fil === item.type))
+    )
+  }
+
+  // Ideally this would be something we could figure out above but I couldn't get a boolean | Function return type to play well with the filter line below
```

Comment:
@skellock maybe you have an idea here? Basically I wanted to make the command optional. If it was included I would evaluate to the boolean, if it was excluded I would return the function that you passed the command to which would then evaluate it. I got TypeScript to be happy about the function definition but the filter below on line 25 threw a fit because it didn't like the possibility of getting a boolean from the function call...